### PR TITLE
`/markdown` read embeds

### LINKED ISSUE #8: Read embeds with /markdown
Currently unsupported feature is reading an embed with the `/markdown` command.

### PR DESCRIPTION
**Pull request by auravoid**
Resolves #8. 

Todo: 
- [x] Read embed fields 
- [x] Add fields for fields that can be read
- [x] Add reference link (could implemented later, not priority)
- [x] Reformat interaction response (see image below)
   - new reply design
- [x] Add catch if message content is too large to escape
- [x] Fix links and how they're escaped weirdly
- [x] Fix weird bug when requesting markdown from a markdown response

### CODE REVIEW

**Comment by auravoid** [comment]
imagine self assigning

**Comment by FloatingMilkshake** [comment]
thanks github i totally meant to do that

File: Commands/Markdown.cs [hunk 1]
```diff
@@ -127,7 +127,7 @@ public class Markdown : ApplicationCommandModule
                         ? "Embed Content"
                         : $"Embed Content: {MarkdownParser.Parse(embed.Title)}")
                     .WithDescription(embeds[0].Description != null ? MarkdownParser.Parse(embed.Description) : "")
-                    .WithColor(embed.Color != null ? embed.Color.Value : Program.BotColor);
+                    .WithColor(embed.Color.Value);
```
Comment (auravoid) [inline Commands/Markdown.cs]: It wasn't unnecessary, just poorly implemented.

File: Commands/Markdown.cs [hunk 2]
```diff
@@ -129,8 +129,8 @@ public class Markdown : ApplicationCommandModule
                         ? "Embed Content"
                         : $"Embed Content: {MarkdownParser.Parse(embed.Title)}")
                     .WithDescription(embeds[0].Description != null ? MarkdownParser.Parse(embed.Description) : "")
-                    .WithColor(embed.Color.Value);
                 
+                    .WithColor(embed.Color.HasValue == false ? Program.BotColor : embed.Color.Value);
```
Comment (auravoid) [inline Commands/Markdown.cs:132]: Ok Github. Move it to a new line.

File: Commands/Markdown.cs (hunk 1 above)
Comment (FloatingMilkshake) [reply Commands/Markdown.cs]: Yeah fair enough, my change there was a bit rushed

**Comment by auravoid** [comment]
Some day.

**Comment by FloatingMilkshake** [comment]
yeah yeah

**Review by FloatingMilkshake — APPROVED** [review]
ty for the PR, much appreciated

**Pull request merged**